loops-and-arrays.js

```diff
@@ -22,6 +26,14 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // ==========================================
 
 const numbers = [2, 4, 5, 29, 38];
+for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 == 0){
+        console.log(numbers[i] * 2);
+    }
+    else{
+        console.log(numbers[i] * 3);
+    }
```

Comment:
if (i % 2 !== 0) {
    numbers[i] = numbers[i] * 3;
  }
  // als je alleen de uitkomst logt (zoals hieronder) zou je de array niet veranderen, en dat wil je juist wel!
  // Dit is dus niet de juiste aanpak:
  // console.log(numbers[i] * 3);

Deze had ik zelf blijkbaar ook niet goed

Reply:
ik zag het ja! wat stom haha